README.md

```diff
@@ -60,6 +60,23 @@ You can also manage apps that on Parse.com from the same dashboard. In your conf
 }
 ```
 
+Parse Dashboard also supports adding an optional icon for each app, so you can identify them easier in the list. To do so, you will need to define an `iconsFolder` in the config file, and define the `iconName` parameter for each app (including the extension). This folder needs to be located in the root of your project. For example, if you are building from this repo's source, the folder should be inside `Parse-Dashboard`; if you are using npm, it should be in your root directory. If no icon is set for a specific app, the dashboard will show a placeholder.
+
```

Comment:
I think the docs should probably talk about 2 use cases: installing globally with `npm install -g parse-dashboard` and installing as part of a project and mounting as an express app. Installing from source is now only recommended if you are working on the dashboard so I don't think we need to call out that use case in the docs.